### ISSUE TITLE
com.reactnativemultithreading.MultithreadingPackage not found

### ISSUE THREAD

**Issue by sitaryo** [mrousavy/react-native-multithreading#22]
hi, I fonund a problom when I try to install react-native-multithreading in my android app.

here is my installation step

1. yarn add react-native-reanimated@next
2. add babel plugin , enable hermes  and add code to MainApplication.class follow the [react-native-reanimated installation](https://docs.swmansion.com/react-native-reanimated/docs/installation/)
3. yarn add react-native-multithreading
4. add code to MainApplication.class follow the [the setup gudie](https://github.com/mrousavy/react-native-multithreading/blob/master/SETUP.md)
5. cd android && ./gradlew clean 
6. cd .. && yarn android

the package version I use
```
"react-native-multithreading": "^1.1.0",
"react-native-reanimated": "^2.0.0",
```

and then I got the error

![image](https://user-images.githubusercontent.com/21635053/118759767-351a5e80-b8a4-11eb-8682-5ab9f9ca8f38.png)

I found that It is no package named `com.reactnativemultithreading` in `node_modules/react-native-multithreading/android`

![image](https://user-images.githubusercontent.com/21635053/118759898-6eeb6500-b8a4-11eb-8181-9ccf330b900f.png)

how can I install the package and use It in android ?

**Comment by mrousavy** [comment]
Try to follow [this issue](https://github.com/mrousavy/react-native-mmkv/issues/37), but for multithreading instead of mmkv.

**Comment by mrousavy** [comment]
it should be supported on 0.63.4 - could you try cleaning your cache?

**Comment by sitaryo** [comment]
I had try `./gradlew clean` before `react-native run-android`,  but not work.

the problem [library "libreactnativeutilsjni.so" not found](https://github.com/mrousavy/react-native-multithreading/issues/22#issuecomment-843908560) had been solved since I upgrade RN version to 0.64.1. thank you for your response 😁
